app/views/shelters/edit.html.erb

```diff
@@ -0,0 +1,16 @@
+<h1>Update Shelter</h1>
+<form class="" action="/shelters/<%= @shelter.id %>" method="post">
+  <input type="hidden" name="authenticity_token" value="<%= form_authenticity_token %>">
+  <input type="hidden" name="_method" value="PATCH"/>
+  <label for="name">Name: </label>
+  <input type="text" name="shelter[name]" id="name" value=" <%= @shelter.name %>"><br>
+  <label for="address">Address: </label>
+  <input type="text" name="shelter[address]" id="address" value=" <%= @shelter.address %>"><br>
+  <label for="city">City: </label>
+  <input type="text" name="shelter[city]" id="city" value=" <%= @shelter.city %>"><br>
+  <label for="state">Sate: </label>
+  <input type="text" name="shelter[state]" id="state" value=" <%= @shelter.state %>"><br>
+  <label for="zip">Zip: </label>
+  <input type="text" name="shelter[zip]" id="zip" value=" <%= @shelter.zip %>"><br>
+  <input type="submit" id="Submit"/>
+</form>
```

Comment:
HTML and CSS are relatively new to me. I would like to improve the style of this page and find out if there is anything I am doing here which is unconventional.